tests/usage_test.py

```diff
@@ -58,6 +58,9 @@ def test_no_args(self):
                         Separate files with commas
   --dont-prepend-root   In project root e.g. /app, imports are not prepended
                         with app.*
+  --no-local-imports    If not set, allow imports of files in the current
```

Comment:
Nit: Double negative feels slightly weird, maybe: `If not set, allow` -> `If set, disallows`